Dockerfile

```
@@ -0,0 +1,7 @@
+FROM openjdk:21-jdk
+
+WORKDIR /app
+
+COPY ./build/libs/rentcar-0.0.1-SNAPSHOT.jar ./rentcar.jar
```

```suggestion
COPY ./build/libs/rentcar-*.jar ./rentcar.jar
```

Please, copy by pattern. Sometimes we update the apps version